tensorflow/go/graph.go

```diff
@@ -92,14 +92,20 @@ func (g *Graph) WriteTo(w io.Writer) (int64, error) {
 // another Graph into g.
 //
 // Names of imported nodes will be prefixed with prefix.
-func (g *Graph) Import(def []byte, prefix string) error {
+func (g *Graph) ImportWithDevice(def []byte, prefix string, device string) error {
```

Comment:
Should we use the options pattern here instead of a new function? That way we can avoid a combinatorial explosion of functions as more options are added (and these are truly optional).

See for example how we do optional attributes in https://godoc.org/github.com/tensorflow/tensorflow/tensorflow/go/op#AvgPool3DAttr

Or alternatively, have a struct of options:

```go
type GraphImportOptions struct {
  Prefix string
  Device string
  // Other things corresponding to TF_ImportGraphDefOptions, but don't need to add them in this PR
}

func (g *Graph) ImportWithOptions(def []byte, options GraphImportOptions) error {
  ...
}

func (g *Graph) Import(def []byte, prefix string) error {
  return g.ImportWithOptions(def, GraphImportOptions{Prefix: prefix})
}
```